hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/EventCategory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.event;
+
+/**
+ * Every Event has a category to help identify what it represents.  It can be one of these, or it can be something
+ * specified by the Event creator.  This just tries to predefine common events.
+ *
+ * @author jay shaughnessy
+ * @author lucas ponce
+ */
+public enum EventCategory {
+    ALERT, DEPLOYMENT, LOG, TRIGGER
```

Comment:
I can identify ALERT, DEPLOYMENT and LOG events. But not sure if I get the semantic of a TRIGGER Event.
Is intended for those Events created internally from a TRIGGER ? right ?